### ISSUE TITLE
reportMessage doestn't get sent after the device reconnects to network.

### ISSUE THREAD

**Issue by chien** [rollbar/rollbar-android#30]
Steps to reproduce:
1. Turn off network on the device
2. Send a message using reportMessage
3. Turn the network on

the message doesn't show up in rollbar.

The same steps work for reportException call.

**Comment by brianr** [comment]
Thanks for the report @chien ! We're planning to work on this soon.

**Comment by brianr** [comment]
Some investigation into this...

The following situation is handled:

1. Turn off the network on the device
2. Send a message using reportMessage. Wait ~15 seconds for the request to fail. You'll see something like the following in logcat:
    
    ```
I/Rollbar: Sending item batch...
I/Basic Network Demo: No wireless or mobile connection.
E/Rollbar: There was a problem reporting to Rollbar.
E/Rollbar: Response: java.net.UnknownHostException: Unable to resolve host "api.rollbar.com": No address associated with hostname
D/Rollbar: Writing items...
D/Rollbar: Items written
    ````
3. Exit the app
4. Turn on the network on the device
5. Open the app

**Comment by ezarowny** [comment]
Hey @chien I'm currently working on this. I should have something for you to test soon.

**Comment by ezarowny** [comment]
@chien could you take a look at #31 when you have a moment?

**Comment by ezarowny** [comment]
Resolved in #31
